src/03-control-flow.js

```diff
@@ -45,27 +49,48 @@ function findLarger(a, b) {}
  *  findLargerOrTie(0, 0);
  *  //> "tie"
  */
-function findLargerOrTie(a, b) {}
+function findLargerOrTie(a, b) {
+  return Math.max(a, b) | "tie"
+
+}
```

Comment:
Would be clearer with an `if` check for equality, I think. Can you explain the `|` operator? Awesome if you've got that down, but I wouldn't use it if you don't!